### PR TITLE
update permissions.yml added doctor fate perms

### PR DESCRIPTION
**Pull request by ZargonKing**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -2928,6 +2928,32 @@ groups:
       suffix: '&9'
     inheritance:
     - mortal
+     worlds:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: too many spaces

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -2928,6 +2928,32 @@ groups:
       suffix: '&9'
     inheritance:
     - mortal
+    worlds:
+      survival:
+        permissions:
+         - ch.alias.empower
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is missing from powers

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -2928,6 +2928,32 @@ groups:
...
+         - essentials.warps.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is missing in powers list

File: PermissionsEx/permissions.yml [hunk 4]
```diff
@@ -2928,6 +2928,32 @@ groups:
...
+         - essentials.back
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is missing in powers list

File: PermissionsEx/permissions.yml [hunk 5]
```diff
@@ -2928,6 +2928,32 @@ groups:
...
+         - -essentials.protect.damage.fall
+         - essentials.tppos
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: this is missing in powers list

**Comment by ZargonKing** [comment]
Doctor Fate Proofs:

Strength of Doctor Fate:
http://i.imgur.com/MsCv1wl.jpg
http://i.imgur.com/f7mGR6a.jpg
http://i.imgur.com/W4X1mBG.jpg

Durability:
http://i.imgur.com/rNx6FQc.jpg
http://i.imgur.com/K1r7WqO.jpg
http://i.imgur.com/hDQNYSi.jpg
http://i.imgur.com/dwJ73Gw.jpg
http://i.imgur.com/8Jel6mK.jpg
http://i.imgur.com/Ppme9Ie.jpg
http://i.imgur.com/LdUcVxg.jpg
http://i.imgur.com/puWX67q.jpg
http://i.imgur.com/Bhltj8N.jpg
http://i.imgur.com/ERqNheL.jpg
http://i.imgur.com/HXc3wrQ.jpg

Speed:
http://i.imgur.com/wuSFwpu.jpg
http://imgur.com/a/KVKIy#0
http://i.imgur.com/AJsh4zq.jpg

Matter Manipulation:
http://i.imgur.com/fWHzVDf.jpg
http://i.imgur.com/S9iyf9B.jpg
http://i.imgur.com/9S7cin1.jpg
http://i.imgur.com/5df3ehd.jpg
http://i.imgur.com/zq8X56I.jpg
http://i.imgur.com/L5An8Hx.jpg

Telepathy/Telekenesis:
http://i.imgur.com/BSM7DsZ.jpg
http://i.imgur.com/K4b9Kkc.jpg
http://i.imgur.com/q3sJslM.jpg
http://i.imgur.com/w0e4UmM.jpg
http://i.imgur.com/NMwsSpC.jpg
http://i.imgur.com/bTaMDUY.jpg
http://i.imgur.com/y5QswVF.jpg
http://i.imgur.com/IUbLksM.jpg

Able to summon and banish:
http://i.imgur.com/puJd8gb.jpg
http://i.imgur.com/JpBynNq.jpg
http://i.imgur.com/zwOXkTd.jpg
http://i.imgur.com/L24umca.jpg
http://imgur.com/a/gcUis
http://i.imgur.com/zZSBFar.jpg

Able to manipulate time:
http://i.imgur.com/jYtmMRN.jpg
http://i.imgur.com/TfyqIdy.jpg
http://i.imgur.com/ilboskO.jpg
http://i.imgur.com/ys2Oywi.jpg

Able to do Interdimensional and Universal Travel:
http://i.imgur.com/VfEUcbn.jpg
http://i.imgur.com/ErH12hG.jpg
http://i.imgur.com/S8yCxK6.jpg
http://i.imgur.com/OzvDPzR.jpg
http://i.imgur.com/WRt0QIS.jpg
http://i.imgur.com/3Gzi8au.jpg

Able to command weather:
http://i.imgur.com/2QK1Zx6.jpg

Able to tap into creation:
http://i.imgur.com/oOgZ5g3.jpg

Able to empower others but was prevented to do so:
http://i.imgur.com/le8bUAL.jpg

**Pull request closed without merging**